cndocs/running-on-device.md

```
@@ -260,78 +365,88 @@ $ npx react-native run-android
+如果这是第一次在 iOS 设备上运行应用，你可能需要注册设备用于开发。从 Xcode 菜单栏打开 **Product** 菜单，然后前往 **Destination**。从列表中查找并选择你的设备。Xcode 将注册你的设备用于开发。
 
 ### 2. 配置代码签名
 
-如果没有[Apple developer account](https://developer.apple.com/)，先注册。
+如果还没有 [Apple 开发者账号](https://developer.apple.com/)，先注册一个。
 
-在 Xcode Project 导航中选择 project，然后选择 main target（它应该和 project 共享同样的名字）。查找"General"标签。前往"Signing"并确保在"Team"下拉下选择了开发者账号或团队。tests target（以 Tests 结尾，在 main target 下面）也需要重复同样的操作。
+在 Xcode Project 导航中选择你的 project，然后选择 main target（它应该和 project 同名）。查找"General"标签，前往"Signing"并确保在"Team"下拉菜单下选择了你的 Apple 开发者账号或团队。对 tests target（名称以 Tests 结尾，在 main target 下面）也**重复**同样的操作。
 
-![](assets/RunningOnDeviceCodeSigning.png)
+![](/docs/assets/RunningOnDeviceCodeSigning.png)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)